ledger/internal/eval.go

```diff
@@ -618,6 +619,7 @@ type LedgerForEvaluator interface {
 	GenesisProto() config.ConsensusParams
 	LatestTotals() (basics.Round, ledgercore.AccountTotals, error)
 	CompactCertVoters(basics.Round) (*ledgercore.VotersForRound, error)
+	MaxAcctLookback() uint64
```

Comment:
you see, it looks really weird having MaxAcctLookback in eval interface. Evaluator should be care